core/digital-twin-lib/src/interfaces/DigitalTwinInterface.ts

```diff
@@ -1,4 +1,4 @@
-export interface DigitalTwin {
+export interface DigitalTwinInterface {
```

Comment:
You could also use the `DigitalTwin` class for this. https://github.com/evannetwork/api-blockchain-core/blob/develop/src/contracts/digital-twin/digital-twin.ts#L133